libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -70,8 +90,26 @@ $button-width: (
   --kirby-button-color: #{utils.get-color('black')};
   --kirby-button-border-color: #{utils.get-color('medium')};
 
+  // NOTE: This won't work!
+  //
+  // @include _button-hover(transparent, 'primary-contrast');
+  //
+  // Because transparent is not defined as a named color variant.
+  // Use the apply-hover() mixin directly instead.
```

Comment:
https://github.com/kirbydesign/designsystem/pull/1991#discussion_r792409627

Regarding the loudness scale. I agree that the names should not be hostile 🙂

The different levels _has_ to have _some_ name and value, but it's not important what they are for now. That will be a UX decision. Since they haven't defined a loudness scale yet we can't use actual names and values. On the other hand, there is no need to stall development while we're waiting for their decision.

I'll change the names of the loudness levels now, but it seems a bit redundant. They could be named `marble`, `golf-ball`, `tennis-ball`, `soccer-ball` etc. - it doesn't really matter. They must be renamed anyway later. We don't know how many we need either. Probably 4-6 levels, but that's also a UX decision we're awaiting 🙂

So, to sum it up: We can't not use names and values for the loudness scale, because technically we need the scale. And we can't use the correct names and values, because they haven't been defined yet.